README.md

```diff
@@ -59,3 +59,11 @@ project were possible, but this project is taking a slightly different direction
 - Based on AdminLTE 3: https://adminlte.io/
 - Using Bootstrap 4: https://getbootstrap.com/
 - Using Font Awesome 5: https://fontawesome.com/
+
+
+
+TODO:
+    - whitenoise in test app, and gunicorn and postgres
+    - add cron for reset
+    - keep it clean
```

Comment:
Is this still a TODO? I just saw you got them installed already. Can't see `DATABASE_URL` but I guess it's postgres?

Reply:
on the heroku app its set, yes it is postgres :)

Ive updated the docs with the link to the app

Reply:
Cool beans!